### PR TITLE
Revert ubuntu 18 azure bridge mitigation

### PR DESCRIPTION
**Pull request by paulgmiller**
Azure cni is in transparent mode since 1.2 which finished releasing o…n 12/15 so this bridge check/manipulation should no longer be needed

### CODE REVIEW

**Comment by paulgmiller** [comment]
@zhedahht @matt-stam

File: parts/linux/cloud-init/artifacts/kubelet.service [hunk 1]
```diff
@@ -18,9 +18,6 @@ ExecStartPre=/bin/mount --make-shared /var/lib/kubelet
 ExecStartPre=-/sbin/ebtables -t nat --list
 ExecStartPre=-/sbin/iptables -t nat --numeric --list
 
-{{/* This is a workaround to setup azure0 bridge for CNI */}}
-ExecStartPre=/usr/local/bin/configure_azure0.sh
```
Comment (weng271190436) [inline parts/linux/cloud-init/artifacts/kubelet.service:22]: after making these changes I think you also need to regenerate files by running make generate

Comment (paulgmiller) [reply parts/linux/cloud-init/artifacts/kubelet.service:22]: Looks like you already did that so lets just use yours. https://github.com/Azure/AgentBaker/pull/512

**Comment by paulgmiller** [comment]
Using wei's https://github.com/Azure/AgentBaker/pull/512

**Pull request closed without merging**